src/tools/run-make-support/src/lib.rs

```diff
@@ -1,16 +1,28 @@
 use std::env;
 use std::path::{Path, PathBuf};
 use std::process::{Command, Output};
-
+mod rustdoc;
+pub use rustdoc::rustdoc;
 pub use wasmparser;
 
+pub fn add_host_rpath_env(cmd: &mut Command) {
+    let ld_lib_path_envvar = env::var("LD_LIB_PATH_ENVVAR").unwrap();
+    let ld_lib_path_value = env::var(&ld_lib_path_envvar).unwrap();
+
+    let temp = env::var("TMPDIR").unwrap();
+    let host_rpath_dir = env::var("HOST_RPATH_DIR").unwrap();
```

Comment:
These should all probably use var_os, in combination with join_paths that shouldn't be too hard.